streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -54,6 +54,7 @@ public RecordCollector recordCollector() {
         return this.collector;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Yes. We throw deprecated `TopologyBuilderException` and should suppress the warning until we resolve https://issues.apache.org/jira/browse/KAFKA-5660